include/config.h

```diff
@@ -46,6 +47,21 @@ struct sway_mode {
 };
 
 /**
+ * libinput options for input devices
+ */
+struct input_config {
+	char *identifier;
+	int click_method;
+	int drag_lock;
+	int dwt;
+	int middle_emulation;
+	int natural_scroll;
+	int scroll_method;
+	int send_events;
+	int tap;
+};
```

Comment:
Can you add (but not necessarily support yet) a boolean for whether or not input from this device can be captured by a window, and a wlc_geometry for the usable region of the screen this input gets? cc @Cloudef, head's up, I'd like those features in wlc